web/yo/app/scripts/controllers/gene.js

```diff
@@ -1647,10 +1644,19 @@ angular.module('oncokbApp')
                 }
                 // review mode check
                 if (mutationReview && mutationReview.get('removed') || tumorReview && tumorReview.get('removed') || treatmentReview && treatmentReview.get('removed')) {
+                    if(reviewObj && !reviewObj.get('removedItem')) {
```

Comment:
Is this a new status `removedItem`? How many statuses do we have now? Is there a document to explain on each status? We may soon lose the track of all statuses.